apps/src/lib/levelbuilder/script-editor/ScriptEditor.jsx

```diff
@@ -711,6 +714,24 @@ class ScriptEditor extends React.Component {
                   </p>
                 </HelpTip>
               </label>
+              <label>
+                Deprecated
+                <input
+                  type="checkbox"
+                  checked={this.state.deprecated}
+                  style={styles.checkbox}
+                  onChange={() =>
+                    this.setState({deprecated: !this.state.deprecated})
+                  }
+                />
+                <HelpTip>
+                  <p>
+                    Used only for Deeper Learning Courses. Prevents Peer Review
```

Comment:
James made a good point yesterday about being consistent in language when talking about Deeper Learning/Professional Learning/Peer Review land, which I'm still struggling to do but I'll do my best here...

"Used only for Professional Learning Courses. Deprecation prevents Peer Reviews conducted as part of this Script from being displayed in the admin-only Peer Review Dashboard."

I'd cut anything about future use (can be updated if/when that happens).

Reply:
Maybe check with Andrea/Tess, they may have better ideas :)

Reply:
Love this update! I'll double check this language is good.

Reply:
Approved and updating now.